src/ducks/resource.js

```diff
@@ -0,0 +1,107 @@
+import apiClient from 'panoptes-client/lib/api-client';
+
+// Action Types
+export const FETCH_RESOURCE = 'FETCH_RESOURCE';
+export const FETCH_RESOURCE_SUCCESS = 'FETCH_RESOURCE_SUCCESS';
+export const FETCH_RESOURCE_ERROR = 'FETCH_RESOURCE_ERROR';
+export const CREATE_TRANSLATION_SUCCESS = 'CREATE_TRANSLATION_SUCCESS';
+
+// Reducer
+const initialState = {
+  original: [],
+  translation: null,
+  error: false,
+  loading: false,
+};
+
+const resourceReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_RESOURCE:
+      return Object.assign({}, initialState, { loading: true });
+    case FETCH_RESOURCE_SUCCESS:
+      return Object.assign({}, state, { original: action.payload, loading: false });
```

Comment:
Yeah, my aim is to eventually store the original here, maybe split the translations out into their own array.

At the moment, the API query will bring back everything. We can either update the query so that it only fetches the original resource, or split the response into the original + translations.

`translation`, by the way, is the translation that you're currently working on, so a single resource.